src/interactive/widgets/footer.rs

```diff
@@ -39,12 +40,12 @@ impl Footer {
             Span::from(format!(
                 "Sort mode: {}  Total disk usage: {}  Items: {} {progress}  ",
                 match sort_mode {
-                    SortMode::SizeAscending => "size ascending",
-                    SortMode::SizeDescending => "size descending",
-                    SortMode::MTimeAscending => "modified ascending",
-                    SortMode::MTimeDescending => "modified descending",
-                    SortMode::CountAscending => "items ascending",
-                    SortMode::CountDescending => "items descending",
+                    SizeAscending => "size ascending",
+                    SizeDescending => "size descending",
+                    MTimeAscending => "modified ascending",
+                    MTimeDescending => "modified descending",
+                    CountAscending => "items ascending",
+                    CountDescending => "items descending",
```

Comment:
This unfortunately is not recommended because it throws away some of the safety features of enums inside match expressions. See: https://www.youtube.com/watch?v=8j_FbjiowvE

```suggestion
                    E::SizeAscending => "size ascending",
                    E::SizeDescending => "size descending",
                    E::MTimeAscending => "modified ascending",
                    E::MTimeDescending => "modified descending",
                    E::CountAscending => "items ascending",
                    E::CountDescending => "items descending",
```

Reply:
Thanks for this hint! I wasn't really seeing it this way just yet, but totally see the point (i.e. as enums are changing their variant names, there is then catch-all variants that show up as warnings for 'unused variables').